setup.py

```diff
@@ -15,8 +16,10 @@ def get_contents(filename):
     return out
 
 #Make Home Directory
-if not os.path.exists(HOME_DIR):
-    os.makedirs(HOME_DIR)
+if os.path.exists(HOME_DIR):
+    shutil.rmtree(HOME_DIR)
+
```

Comment:
Thats useful for debugging purposes.When you install UIP for first time settings.json file will be saved in .uip folder but after when you change any option like timeout manually in code and reinstall it wasn't being overwritten don't know why that was happening.So, I am just deleting the settings of any previous installation if available.